cmd/test.go

```diff
@@ -163,6 +180,10 @@ func opaTest(args []string) int {
 	var coverTracer topdown.Tracer
 
 	if testParams.coverage {
+		if testParams.benchmark {
+			fmt.Fprintln(os.Stderr, "unable to report coverage for benchmarks")
```

Comment:
nit: This error message feels a bit misleading--could we indicate in a concise way, why coverage and benchmarks are not allowed together?

Reply:
Is it misleading? It just isn't supported, so OPA is unable to report it for benchmarks. I'll update to "coverage reporting is not supported when benchmarking tests" which is maybe less ambiguous.

It is something that could be added on but it's unclear how best to do it and make it useful.